tests/integration/StorageFileTest.php

```diff
@@ -19,7 +19,10 @@ public function setup(): void
 		$api_key = getenv('API_KEY');
 		$reference_id = getenv('REFERENCE_ID');
 		$bucket_id = 'test-bucket';
-		$this->client = new  \Supabase\Storage\StorageFile($api_key, $reference_id, $bucket_id);
+		$scheme = 'https';
```

Comment:
After the __constructor is updated none of these should be needed - please remove them.